pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -338,6 +358,34 @@ func (f *framework) RunScorePlugins(pc *PluginContext, pod *v1.Pod, nodes []*v1.
 	return pluginToNodeScoreMap, nil
 }
 
+// RunNormalizeScorePlugins runs the normalize score plugins. It should be called after
+// RunScorePlugins with the PluginToNodeScoreMap result. It then modifies the map with
+// normalized scores. It returns a non-success Status if any of the normalize score plugins
+// returns a non-success status.
+func (f *framework) RunNormalizeScorePlugins(pc *PluginContext, pod *v1.Pod, scores PluginToNodeScoreMap) *Status {
+	ctx, cancel := context.WithCancel(context.Background())
+	errCh := schedutil.NewErrorChannel()
+	workqueue.ParallelizeUntil(ctx, 16, len(f.normalizeScorePlugins), func(index int) {
+		pl := f.normalizeScorePlugins[index]
+		// During framework initialization, we make sure that NormalizeScore plugins are
+		// a subset of Score plugins. So nodeScoreList must always exist.
+		nodeScoreList, _ := scores[pl.Name()]
```

Comment:
We should still test that it actually exist and throw an internal error, it may help us debug bugs (e.g., RunScorePlugins not adding nodeScoreList at all, or Name() return something different since it is still a function).

Reply:
SGTM.